src/myst.ts

```diff
@@ -36,6 +36,28 @@ import { imageUrlSourceTransform } from './images';
 import { internalLinksPlugin } from './links';
 import { addCiteChildrenPlugin } from './citations';
 
+export interface MySTOptions {
+
+  parserOptions: Partial<AllOptions>;
+}
```

Comment:
@fwkoch is there any thinking we need to on this before we start encouraging using these more widely?!

They look pretty solid I think.